src/WebJobs.Script.Grpc/Abstractions/IRpcServer.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Script.Abstractions.Rpc
+{
+    public interface IRpcServer
```

Comment:
nit: Can you doc this contract more? Suggestions below.

Reply:
Agreed, will add more docs regarding semantics.